### ISSUE TITLE
filter reference list and display as table on reference tab

### ISSUE THREAD

**Issue by LesleyAtwood** [Science-for-Nature-and-People/Midwest-Agriculture-Synthesis#34]
Use paper_id lists in summary table to filter reference list and display it on the reference tab. 

When the user clicks back to the figure will it retain the selected filters?

**Comment by nathanhwangbo** [comment]
Just added this!
The plot filters should be retained when you switch between tabs

**Comment by swood-ecology** [comment]
Awesome. A couple more issues here:

1. I think the short_id column can be dropped from the table
2. Can the DOI by a hyperlink?
3. Can there be a column that shows which of the treatments and outcomes are present in each paper?

**Comment by LesleyAtwood** [comment]
Currently the reference list doesn't include treatment and outcome information, which may make #3 a little challenging. 

If I were to update the reference list to include that information what would be the best way to do it? With the caveat that most papers have more than one response variable [`outcome`]. Use a many-to-one approach so we filter specific rows? This could create redundancies if the user views more than one `outcome` at a time

**Comment by swood-ecology** [comment]
I don't think you would have to change how the filtering is done. I'd just suggest having a column that has comma-separated practices and another that has comma-separated outcomes that get visualized.

**Comment by LesleyAtwood** [comment]
The reference list is currently filtered by `Paper_Id` only. 

With a comma-separated list of practices and outcomes, the end result would be list of practices and outcomes included in each paper. I like the idea of this, but it may be overwhelming to have an exhaustive list of all the outcomes for each paper. Plus it won't be as easy to filter/organize the output so it logically matches the figure. Separate columns would be needed for that.

**Comment by swood-ecology** [comment]
How would you approach it?

**Comment by LesleyAtwood** [comment]
Here's a possible solution:

We convert the reference table to widget table using `renderDataTable({df()})`. This widget includes clickable sorting features and a search box. 

Then we can use some feature (`detect_stri`, `arrange` ??) to reorder the comma-delimited list of `outcomes` so the most relevant `outcome`(s) are displayed first. This should make it easier to sort.
